src/components/Avatar.js

```diff
@@ -10,13 +10,17 @@ const propTypes = {
     /** Extra styles to pass */
     style: PropTypes.arrayOf(PropTypes.any),
 
+    /** Extra styles to pass to View wrapper */
+    containerStyle: PropTypes.arrayOf(PropTypes.any),
```

Comment:
Can this be `containerStyles` since we are accepting an array of style objects?

Also should be `PropTypes.arrayOf(PropTypes.object)`

Reply:
>Named the prop containerStyle instead of containerStyles to be more in line with similar prop names across the app. Also made sure the new wrapper style is always there no matter what you pass with containerStyle.

There is a guide here on the correct usage -> https://github.com/Expensify/Expensify.cash/blob/main/STYLING.md#complex-component

Probably we need to update some other incorrect usages :)

Reply:
There are at least ten props in the app that end with `style` (not `styles`) and most of them accept `any`. Do I change all the ones I could find? The way I see it, it's outside the scope of the issue, especially assuming I am expected to test each change.

![DeepinScreenshot_select-area_20210526211617](https://user-images.githubusercontent.com/64093836/119710952-a3908b00-be67-11eb-927c-4364adf2771d.png)

I'll change the naming and proptype for the new `containerStyle` prop and I guess the existing `style` prop of `Avatar` component too, if that's ok.

Reply:
No, please only change this specific instance.

If we see a `style` prop that means it can accept either an array or object. It's explained [here](https://github.com/Expensify/Expensify.cash/blob/main/STYLING.md#simple-component).

Reply:
Ok, does it mean that proptype should be
```js
    /** Additional styles to add after local styles */
    style: PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.object),
        PropTypes.object,
    ]),
```

instead of `PropTypes.arrayOf(PropTypes.object),`

Reply:
No, because this is not a `style` prop for a simple component, but specifically targets the wrapper element so we are looking for:

```js
containerStyles: PropTypes.arrayOf(PropTypes.object)
```

Let me know if that makes sense or if the readme can be clearer about this somehow.

Reply:
No, I mean the `style` prop, not `containerStyles`.

It was `style: PropTypes.arrayOf(PropTypes.any),` before I touched it, my question is should it be 
```js
    style: PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.object),
        PropTypes.object,
    ]),
 ```
 
 Since it's a simple style prop it should accept both object and array, as per styling guide.

Reply:
Pushed an update. The way I see it, it's either like this or we rename the `style` prop to something like `imageStyles` and allow only array of objects there. If I understand the [styling guidelines](https://github.com/Expensify/Expensify.cash/blob/main/STYLING.md#complex-component) correctly, this how it is meant to be if you pass 2 style props: all of them should be plural, must have names describing the target element and accept only array of objects.

Reply:
Yes, but we should not have a combination of `style` and `containerStyles` either we have multiple elements and we are getting specific or a single element and we are being vague. 

It's totally fine if you don't want to make the change. We have some clean up to do and are still figuring some things out.